packages/cli/src/lib/index.js

```diff
@@ -50,6 +50,34 @@ export const getSchema = async (name) => {
   return metadata.datamodel
 }
 
+/**
+ * Returns the enum defined with the given `name` parsed from
+ * the schema.prisma of the target applicaiton. If no `name` is given then the
+ * all enum definitions are returned
+ */
+export const getEnum = async (name) => {
```

Comment:
This method is almost an exact copy of `getSchema()`... maybe `getSchema()` accepts an optional parameter that says what datatype it should return (`model` or `enum`). Then it's expandable for any other future data type that prisma adds? Leaving it blank could default to `model` so no existing code needs to change?

Reply:
I refactored this slight extracting the method call to `getDMMF`. Have a look.

The main reason for not going with the optional parameter is the return type that would be specified if this was in Typescript. It would have to be `Model[] | DatamodelEnum[]`, which I found confusing.